src/Storefront/Pagelet/Menu/Offcanvas/MenuOffcanvasPagelet.php

```diff
@@ -3,14 +3,22 @@
 namespace Shopware\Storefront\Pagelet\Menu\Offcanvas;
 
 use Shopware\Core\Content\Category\Tree\Tree;
+use Shopware\Core\Framework\Feature;
 use Shopware\Core\Framework\Log\Package;
 use Shopware\Storefront\Pagelet\NavigationPagelet;
 
 #[Package('framework')]
 class MenuOffcanvasPagelet extends NavigationPagelet
 {
+    /**
+     * @deprecated tag:v6.7.0 - Will be removed, as it is unused
```

Comment:
Do we need the extra class then at all?

Or would it be big hassle to rework & deprecate that to use the NavigationPagelet?

Reply:
I am not sure if it is worth the effort :see_no_evil: it is typed in several places, which would all need a deprecation :grimacing: 

also I think it is not too bad to have a dedicated class for each navigation case.